Assets/MRTK/Examples/Experimental/BoundsControl/Scenes/BoundsControlConfigs/Precision Configs/PrecisionRotationHandlesConfiguration.asset

```diff
@@ -0,0 +1,29 @@
+%YAML 1.1
```

Comment:
This is a SUPER-nit, but would it be possible to rename `Precision Configs` to `PrecisionConfigs`? We've tended to try to not have spaces in folder / file names.